pkg/version/version.go

```diff
@@ -34,26 +33,32 @@ type CiliumVersion struct {
 	GoRuntimeVersion string
 	// Arch is the architecture where Cilium was compiled
 	Arch string
+	// BuildDate is the build time reference stored as string
```

Comment:
Specify the date format used here. I'd recommend using a standard format, specifically RFC 3339 with "T" as separator. That will be unambiguous and trivial to parse.

To change the date/time format to RFC 3339, you'll need to change `Makefile.defs` to call `git show` as:
```
git show -s --format='format:%h %aI'
```
instead of:
```
git show -s --format='format:%h %aD'
```